scopes/workspace/workspace/workspace.ui.runtime.tsx

```diff
@@ -235,7 +235,7 @@ export class WorkspaceUI {
       component: function Gallery() {
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-dd0fc6034d2ab3a73eb49449133831c5bba5463feabd33b3ef40944653037576